rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,292 @@
+hierarchy [figure of how that json is rendered].
+Given the size of many series and Host Item Entry that would not be practical in this case, so the proposal 
+is for the frontend to choose whether to render the hierarchy base on the Work `WorkType`, which is `Collection` 
+in the case of archive Works.
+
+As series and Host Item Entries can have volumes, the proposal is to represent those as Items.
+Additionally, if a serie or Host Item Entry has an identifier (such as ISSN os ISBn or other) the proposal is 
+to mint a canonical identifier in the pipeline and to expose that identifier in the API. 
+Finally, the proposal is to model a serie statement or a Host Item Entry with a type `Work`.
+
+### Examples
+ 
+- [1074785](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1074785?lang=eng&suite=cobalt&marcData=Y) 
+This is an example of a serie statement in 490 + 830. In this example the information contained in 830 and 490 is 
+  exactly the same, so the result is just one statement. 
+```
+"partOf": [
+    {
+        "title": "Morphogenesis of the vertebrate brain",
+        "items": [
+            {
+                "title": "2",
+                "type": "Item",
+                "locations": []
+            }
+        ],
+        "type": "Work"
+    }
+]
+```
+- [3109805](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3109805?lang=eng&suite=cobalt&marcData=Y)
+  490 + 773 with slightly different information
+```
+"partOf": [
+    {   //490
+        "title": "Early European Books : Printed sources to 170",
+        "items": [
+            {
+                "title": "Collection 4",
+                "type": "Item",
+                "locations": []
+            }
+        ],
+        "type": "Work"
+    },
+    {   //773
+        "title": "Early European Books",
+        "type": "Work"
+    },
+]
+```
+
+- [3178769](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3178769?lang=eng&suite=cobalt&marcData=Y) 490 + 830 multiple entries + 773
+```
+"partOf": [
+    {   // 490 & 830 with same information
+        "id": "abcdefg",
+        "identifiers": [
+            {
+                "identifierType": {
+                    "id": "issn",
+                    "label": "ISSN",
+                    "type": "IdentifierType"
+                },
+                "value": "1089-3938",
+                "type": "Identifier"
+            }
+        ],
+        "title": "Perspectives in Continental philosophy series",
+        "type": "Work",
+        "items": [
+            {
+                "title": "no. 39",
+                "type": "Item",
+                "locations": []
+            }
+        ]
+    },
+    {  // 830 with slighly different information
+        "title": "Fordham perspectives in continental philosophy",
+        "type": "Work"
+    },
+    {   // 830 & 773 contain the same information
+        "title": "ACLS Humanities E-Book.",
+        "type": "Work"
+    }
+]
+```
```

Comment:
Comments from @jtweed:
> - What a great example!
> - [Aside: I love how the first series link doesn't even return any results in Encore...]
> - When we create this identified work I'm assuming we're using the ISSN as the source identifier?
> - Items is probably the _better_ way to handle volumes, but for simplicity might be easier to make part of the work title?
> - That might be even more true if this isn't really a volume but is actually potentially part of a hierarchy?
> - Again series fields should probably be Series, if we don't know what the 773 is, Work is fine

Reply:
> When we create this identified work I'm assuming we're using the ISSN as the source identifier?

that would be my plan, yes

Reply:
> Items is probably the better way to handle volumes, but for simplicity might be easier to make part of the work title?
That might be even more true if this isn't really a volume but is actually potentially part of a hierarchy?

The main thing that bothers me is if we use ISSN or whatever to identify them, those refer to the whole Serie or host item entry (773) not to the volumes so you have many bibs referring the same ISSN but the volume they refer to is different. It makes me think that somehow volume should be a separate thing.
 https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y
is an example of that. That issn is used in 490 by 1037 bibs but the volumes are different.

Reply:
> Again series fields should probably be Series, if we don't know what the 773 is, Work is fine

What is the difference between a Serie and a Work though? A Serie can have a ISSN and there is definitely overlap in some instances between 440/490 and 773 but it’s a bit weird if we use different types in those instances? Or do we decide to dedupe in those cases?
I know we said no `workType` cause it should be called Format etc, but couldn’t it be used for Works extracted from 440/490? eg
```
{
...
"workType": "Serie",
"type": "Work:,
 }
 ```

Reply:
I just found out that works from calm can have type `Series`, like [this](https://api.wellcomecollection.org/catalogue/v2/works/dq3spb42). Maybe that is an argument in favour of using type `Series` if they are used in a similar way in calm and sierra?
I guess if we decided to go for type `Series` for things extracted from 440/490 the front end would need to use the `workType` property to decide how to render, instead of `type`.

Reply:
> The main thing that bothers me is if we use ISSN or whatever to identify them, those refer to the whole Serie or host item entry (773) not to the volumes so you have many bibs referring the same ISSN but the volume they refer to is different. It makes me think that somehow volume should be a separate thing.

That is (possibly unfortunately) correct. An ISSN refers to an entire ongoing serial publication, eg a journal, not a specific volume or issue of that publication. So it's not correct for us to assume it refers to a specific volume or part and we shouldn't try to use it to uniquely identify a part.

Reply:
> What is the difference between a Serie and a Work though? A Serie can have a ISSN and there is definitely overlap in some instances between 440/490 and 773 but it’s a bit weird if we use different types in those instances? Or do we decide to dedupe in those cases?

Nothing really, a Series in our model is just a more specific type of Work. It's unlikely to be even rendered on the page as such. It's just an attempt to hint that it came from a series statement rather than a more general relationship, so is likely to be something considered a Series. A work can (and many of ours do) have an ISSN, eg journals. If there's overlap, possibly it's better to just keep them all the more generic Work if we can't say what it really is.

> I know we said no `workType` cause it should be called Format etc, but couldn’t it be used for Works extracted from 440/490? eg
> 
> ```
> {
> ...
> "workType": "Serie",
> "type": "Work:,
>  }
> ```

I'd disagree with this. We use format/workType to describe what Sierra calls the material type, ie it's ephemera, it's a book, it's a journal, it's a manuscript etc. A Series could in theory be any of those formats.

Reply:
> I just found out that works from calm can have type `Series`, like [this](https://api.wellcomecollection.org/catalogue/v2/works/dq3spb42). Maybe that is an argument in favour of using type `Series` if they are used in a similar way in calm and Sierra?

This is why we added the more specific subtypes of Work, to handle the hierarchical relationships in archives and, we hoped, to help model library series statements in a similar way.

> I guess if we decided to go for type `Series` for things extracted from 440/490 the front end would need to use the `workType` property to decide how to render, instead of `type`.

Yes, the front end will have to render based on the workType. Eg show archive tree for archives, but not for books as the UI for archives won't handle a single level with tens of thousands of entries. That's why we don't want to denormalise here and instead intend to link to a search results page.

Reply:
> I'd disagree with this. We use format/workType to describe what Sierra calls the material type, ie it's ephemera, it's a book, it's a journal, it's a manuscript etc. A Series could in theory be any of those formats.

Yes, after talking with @jamesgorrie , I see this now